data/tool_pages/platform/2du_00_dune_developer_preview.md

```diff
@@ -0,0 +1,56 @@
+---
+id: "dune-developer-preview"
+title: "Dune Developer Preview"
+description: Description about the goals and the links for the Developer Preview Program
+category: "Dune"
+---
+
+## Context of the Dune Developer Preview
+
+Since 2012, [opam](https://opam.ocaml.org) has been the default system to
+manage package in the OCaml ecosystem. It allows interacting with a central
+package repository called `opam-repository`. It contains around 5000 packages of
+the OCaml ecosystem.
+
+However, with time, the OCaml Community has raised concerned about having a
```

Comment:
I'd lead this with positive things about the new workflow instead of negative things about the existing one.

Reply:
Would it sound better if I replace the "However, with time, [...] have raised concerned" by "With time, the OCaml Comunity have suggested to have [...]"? This way, the sentence does not contain anything negative.